metis-enrichment/metis-enrichment-common/src/main/java/eu/europeana/enrichment/utils/InputValue.java

```diff
@@ -14,7 +14,7 @@
  *  See the Licence for the specific language governing permissions and limitations under
  *  the Licence.
  */
-package eu.europeana.metis.utils;
+package eu.europeana.enrichment.utils;
 
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize.Inclusion;
```

Comment:
Can we get rid if this deprecated reference?

Reply:
Done.